src/main/resources/api/api.yaml

```diff
@@ -150,6 +150,160 @@ paths:
   #
   # Questions
   #
+  /v1/questions:
+    get:
+      summary: Get the list of all questions in the specific topic
+      operationId: listQuestions
+      tags:
+        - questions
+      parameters:
+        - name: topicId
+          in: query
+          required: true
+          description: Id of the specific topic
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: Return an array of the specific topic's questions
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/V1Question'
+        '400':
+          description: Invalid topic ID supplied
+          schema:
+            $ref: '#/definitions/V1Error'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        '404':
+          description: Questions not found
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'
+    post:
+      summary: Add a new question
+      operationId: createQuestion
+      tags:
+        - questions
+      parameters:
+        - name: body
+          in: body
+          required: true
+          description: Question that need to be added
+          schema:
+            $ref: '#/definitions/V1Question'
+      responses:
+        '200':
+          description: Return a new question
+          schema:
+            $ref: '#/definitions/V1Question'
+        '400':
+          description: Invalid input
+          schema:
+            $ref: '#/definitions/V1Error'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'
+  /v1/questions/{questionId}:
+    parameters:
+      - name: questionId
+        in: path
+        required: true
+        description: Id of the question to retrieve
+        type: integer
+        format: int64
+    get:
+      summary: Get the question with a specific id
+      operationId: showQuestionById
+      tags:
+        - questions
+      responses:
+        '200':
+          description: Return the question
+          schema:
+            $ref: '#/definitions/V1Question'
+        '400':
+          description: Invalid question ID supplied
+          schema:
+            $ref: '#/definitions/V1Error'
+        '404':
+          description: Question not found.
+          schema:
+            $ref: '#/definitions/V1Error'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'
+    put:
```

Comment:
Miss at the first time that problem, put must have no path parameters (as in user API), because topic id already coded in question body.
Move that request description into `/v1/questions` path